packages/@expo/metro-config/src/utils/filePath.ts

```diff
@@ -0,0 +1,8 @@
+const REGEXP_REPLACE_SLASHES = /\\/g;
+
+/**
+ * Convert any platform-specific path to a POSIX path.
+ */
+export function toPosixPath(filePath: string): string {
+  return filePath.replace(REGEXP_REPLACE_SLASHES, '/');
+}
```

Comment:
duplicate the function in both cli and metro-config. it doesn't make sense for metro-config to export this utility function for cli use.